doc/users/next_whats_new/hatch-alpha.rst

```diff
@@ -0,0 +1,6 @@
+The SVG and PDF backends can now render hatches with transparency
+-----------------------------------------------------------------
+
+The SVG and PDF backends now respect the hatch stroke alpha. Useful applications
+are, among others, semi-transparent hatches as a subtle way to differentiate
+columns in bar plots.
```

Comment:
This could be a nice demo in the gallery, or perhaps another subplot in https://matplotlib.org/3.2.0/gallery/lines_bars_and_markers/filled_step.html

Reply:
Cool idea. As a side note, because hatch color/alpha is linked to the Patch's edge's, a simple plot with hatch alpha will usually not look so good because the translucent edge will half-overlap with the fill. One can plot edge-less with `lw=0`, but to have an opaque outline one must replot a full opacity, edge-only plot on top of the first.
It's a bit hackish. An independent hatch style API (#7059) would go a long way to simplifying this.